### DISCUSSION TITLE
스킨을 적용 시 페이지가 무한로딩됩니다

### DISCUSSION THREAD

**Discussion by gyull0210** [pronist/hELLO#96] [Q&A] [answered]
메인페이지로 들어가지지 않고 로딩창에서 멈춰있습니다

**Comment by pronist** [comment, marked as answer]
로딩 화면은 **자바스크립트**에 의해 동작을 멈추도록 되어있는데, 만약 ```app.js``` 자바스크립트가 정상적으로 로드 되지 않았다면, 스크립트가 동작하지 않아 계속 로딩 페이지가 나타날 수 있습니다. 로딩이 잘 되었나요?

**Comment by gyull0210** [comment]
알고보니 폴더 안의 파일을 추가로 업로드하지 않아 발생한 일이었습니다. 답변주셔서 감사합니다.

2020년 3월 23일 (월) 오후 4:51, SangWoo Jeong <notifications@github.com>님이 작성:
